system/assets/js/admin/presidecore/formFields.js

```diff
@@ -53,4 +53,28 @@
         sideBySide:true
 	});
 
+
+	var width       = $('.image-dimensions-picker-width');
```

Comment:
This is not quite working for me - in the JS here we have two issues:

1. Using element IDs instead of classes (#derivative_chosen, #derivative) - there could be multiple instances of this in a page potentially - the JS should work for each form independently
2. Refering to classes outside of the context of an individual derivative form instance

ultimately, we need this:

    for each derivative field:
        find its related height, width and quality form fields
        set disabled/enabled status of those fields according to value of derivative field
        when derivative field changes, recalculate disabled/enabled status of those fields
    end for;